core/src/main/java/org/codice/imaging/nitf/core/ImageCoordinates.java

```diff
@@ -34,7 +35,7 @@
 
         @param coord array of four coordinate pairs, in the order [0,0], [0, MaxCol], [MaxRow, MaxCol], [MaxRow, 0].
```

Comment:
Is this comment still appropriate? Should we use something other than an array here?